src/Uno.UWP/System.Runtime/WindowsRuntimeBufferExtensions.cs

```diff
@@ -27,9 +27,24 @@ public static IBuffer AsBuffer(this byte[] source, int offset, int length, int c
 				throw new ArgumentNullException(nameof(source));
 			}
 
-			var buffer = new UwpBuffer((uint)capacity);
-			Array.Copy(source, offset, buffer.Data, 0, length);
-			return buffer;
+			if (offset < 0) throw new ArgumentOutOfRangeException(nameof(offset), "offset is less than 0 (zero).");
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6307767)